src/modules/formBuilder/components/itemEditor/conditionals/EnableWhenCondition.tsx

```diff
@@ -101,18 +102,42 @@ const EnableWhenCondition: React.FC<EnableWhenConditionProps> = ({
     name: `${enableWhenPath}.${index}`,
   });
 
+  const {
+    field: { onChange: onChangeOperator },
+  } = useController({
+    control,
+    name: `${enableWhenPath}.${index}.operator`,
+  });
+
   const dependentItem = useMemo(
     () => (state?.question ? itemMap[state?.question] : undefined),
     [itemMap, state?.question]
   );
+
+  const localPickList = useMemo(() => {
+    if (dependentItem?.pickListReference)
+      return localResolvePickList(dependentItem?.pickListReference);
+  }, [dependentItem]);
+
+  const { pickList: remotePickList, loading: remotePickListLoading } =
+    usePickList({
+      item: {
+        linkId: 'fake',
+        type: ItemType.Choice,
+        pickListReference: dependentItem?.pickListReference,
+      },
+      fetchOptions: {
+        skip:
+          !dependentItem || !dependentItem.pickListReference || !!localPickList,
```

Comment:
I would have thought that we don't need the `!!localPickList` check, nor the `localResolvePickList` call above. The usePickList hook should [detect](https://github.com/greenriver/hmis-frontend/blob/14715dbffd123391bf11d8f77803dd26f6488a0a/src/modules/form/hooks/usePickList.tsx#L80) whether it's a locally resolvable pick list, and if so returns that (And skips the remote call).

Reply:
You are right. This can be simplified a lot! Thank you!

Reply:
I'm not sure if `|| { linkId: 'fake', type: ItemType.Choice }` on line 118 is really the best way to keep typescript happy; maybe `usePickList` should accept item as optional and return early if it's not provided, but I was more hesitant to change a broadly used interface. (Passing `skip` option doesn't work because it skips the network call but not the whole hook, and it also overwrites the `skip` logic internal to that hook, which turns out to be important)

Reply:
Yes I agree, the interface for usePickList could certainly be improved!